101-XSS-Attack-Prevention/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "XSS attack on webapp Scenario",
+    "description": "This will deploy 2 application gateway, 1 webapp, sql server and database, OMS and other network resources. One app gateway is on detection mode and other is on prevention mode. Perform the XXS attack by following the guidlines and execute the scenario for mitigation and prevention of a XXS attack.",
```

Comment:
This will deploy 2 application gatewaya , a web app, a sql server and database, OMS and other network resources. One app gateway is in detection mode and other is in prevention mode. Perform the XSS attack by following the guidleines and execute the scenario for mitigation and prevention of a XSS attack.

Reply:
updated as per recommended